MatrixSDK/Crypto/Data/Store/MXRealmCryptoStore/MXRealmCryptoStore.m

```diff
@@ -24,11 +24,16 @@
 #import "MXSession.h"
 #import "MXTools.h"
 #import "MXCryptoTools.h"
+#import "MXKeyProvider.h"
+#import "MXRawDataKey.h"
 
 NSUInteger const kMXRealmCryptoStoreVersion = 13;
 
-static NSString *const kMXRealmCryptoStoreFolder = @"MXRealmCryptoStore";
+NSString *const kMXRealmCryptoStoreDataType = @"org.matrix.sdk.MXRealmCryptoStoreDataType";
```

Comment:
For external constants it is recommended to not use the `k` prefix: `MXRealmCryptoStoreDataType`